### PR TITLE
Fixes Issue #7655: Immediate notifications had incorrect message

### LINKED ISSUE #7655: Notification mail template is inconsistent with "Immediately" notification setting
#### Description

Continued from https://www.simplemachines.org/community/index.php?topic=584475.0

Even if you set the notification settings to "Immediately" rather than "Immediately - First time only", the mail template says the following:

> More replies may be posted, but you won't receive any more notifications for this topic until you visit it.

I certainly got a bit confused because I got more than one mail, but I also didn't get a mail for every reply. This sentence should probably be omitted from the mail if notifications are set to "Immediate". Maybe it should also mention that not every reply will trigger an immediate mail notification, as SMF tries to avoid being too spammy.

### PR DESCRIPTION
**Pull request by nc3man**
Fixes https://github.com/SimpleMachines/SMF/issues/7655
Notification mail template is inconsistent with "Immediately" notification setting

The value of $frequency must be checked in order to select the correct message based on user notification frequency setting

### CODE REVIEW

**Comment by nc3man** [comment]
Actually, I committed a version I was working with against SMF 2.1.2. I didn't realize CreatePost-Notify.php had considerable 2.1.4 changes. Let me fix that up and re-commit. I only changed one line. 
Sorry about that - my first PR on this project.

File: Sources/tasks/CreatePost-Notify.php [hunk 1]
```diff
@@ -528,8 +528,8 @@ protected function handleWatchedNotifications()
 					(!empty($this->prefs[$member_id]['board_notify']) ? $this->prefs[$member_id]['board_notify'] : 0);
 
 				$content_type = 'board';
-
-				$message_type = !empty($frequency) ? 'notify_boards_once' : 'notify_boards';
+
```
Comment (Sesquipedalian) [inline Sources/tasks/CreatePost-Notify.php]: Please remove these unnecessary whitespace characters.

**Review by Sesquipedalian — CHANGES_REQUESTED** [review]

File: Sources/tasks/CreatePost-Notify.php (hunk 1 above)
Comment (nc3man) [reply Sources/tasks/CreatePost-Notify.php]: Which are unnecessary? Do you want the line prior to the $message_type line removed? It was there before.

For the code on line 532, I was trying to follow the same style that I see on line 528 with white space around both "?" and ":".

Happy to make changes but white space is invisible and it's usually added for clarity.

Comment (nc3man) [reply Sources/tasks/CreatePost-Notify.php]: Got it. I was sloppy by grabbing a whitespace line for copy/paste from my 2.1.2 working version. Latest commit only changes line 532.

**Comment by jdarwood007** [comment]
Can you rebase your PR to remove the extra commits?
You essentially want to delete the commits and then rebase against the latest code.
https://stackoverflow.com/questions/36168839/how-to-remove-commits-from-a-pull-request

**Comment by live627** [comment]
err, then squash all three commits.

**Comment by nc3man** [comment]
I understand the extra commits clutter the history, but I am not very experienced with git. The reference only confused me. Sorry that I bolluxed a simple one line change. Here is what I did. I forked the master SMF branch to my local account. I did not create a branch. Then on my win laptop where I have git bash installed, I simply ran git clone "https://github.com/nc3man/SMF-2.1", made the edit to CreatePost-Notify.php in my local repository by unfortunately using a , then pushed it back up to github. Unfortunately, I uploaded a change

**Comment by nc3man** [comment]
Sorry, I fat fingered something which completely closed the pull request. I was trying to explain what I did. Perhaps it would be best for me to just start over and create a clean pull request. I wanted to dash those 2 extra commits anyway. I apologize for bolluxing a simple change, but I am willing to learn new stuff. How can I just delete this pull request to start over? Can someone do it for me or is there something I can do?
